admin/tabs/cluster/ClusterTabModel.ts

```diff
@@ -57,20 +60,28 @@ export class ClusterTabModel extends HoistModel {
             let data = await XH.fetchJson({url: 'clusterAdmin/allInstances', loadSpec});
             data = data.map(row => ({
                 ...row,
+                isLocal: row.name == XH.environmentService.serverInstance,
                 usedHeapMb: row.memory?.usedHeapMb,
                 usedPctMax: row.memory?.usedPctMax
             }));
 
             gridModel.loadData(data);
             await gridModel.preSelectFirstAsync();
         } catch (e) {
+            gridModel.clear();
             XH.handleException(e);
```

Comment:
This seems like a good candidate for an inline exception message, vs. the modal.  Fast polling + modal exceptions + admin console not an ideal combination.

A component we don't have yet but which seems ideal for this would be some kind of errorMessageBanner. I would love to keep the last-loaded data in the grid, but then show a non-modal, non-grid-replacing, but still-very-obvious alert that the data in that grid failed on last refresh.  Would be interested to know if that's a use case or idea that's come up elsewhere.

For now, we could swap out the grid with our standard errorMessage component to avoid the modal - or if the above sounds interesting, explore it as a one-off in this component to see if we like it.

Reply:
+1 for ther "errorMessageBanner" idea

Reply:
Well that's a very good point! I think the issue with keeping the last-loaded in the grid was that it preserved the grid selection, which the log viewer model was observing to poll for instance logs.  We could clear the selection on error or have the models of child components observe that this model is in an error state, although maybe cleaner to just display the error panel in place of the grid?

Reply:
child comps should have their own reasonable error handling i would think